dylint_lints/README.md

```
@@ -204,6 +208,104 @@ cx.span_lint(MY_LINT, item.span, |diag| {
+```
+
+**Location:** [`validate_gts_docs.py`](validate_gts_docs.py)
+
+**Exit codes:**
+- `0` - All GTS identifiers are valid
+- `1` - Invalid GTS identifiers found (fails CI)
+
+### GTS Identifier Format
+
+A GTS identifier follows this structure:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)